R/get_allele_counts.R

```diff
@@ -597,6 +588,7 @@ get_allele_counts <- function (i , patient_id, sample_id, sex, bam_file, mq=0,
   
   # Create file
   f_nm <- paste(path_output, patient_id, ".", sample_id, ".", i, ".SNPs.CpGs.fst", sep = "")
+  print(f_nm)
```

Comment:
Convert to cat statement with "written to"

Reply:
Or use logger as discussed!

Reply:
Used cat for now!